### PR TITLE
Add ESlint and .env

### PR DESCRIPTION
**Pull request by timmui**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            "allowKeywords": true,
+            "allowPattern": ""
+        }],
+        "dot-location": ["warn", "property"],
+        "brace-style": ["warn", "1tbs"],
+        "camelcase": ["warn", {"properties": "never"}],
+        "comma-style": ["warn", "last"],
+        "comma-dangle": ["warn", "always-multiline"],
+
+
```
Comment (jack-mo) [inline .eslintrc.json]: 👾

**Review by jack-mo — CHANGES_REQUESTED** [review]
LGTM, accept comment

File: .eslintrc.json (hunk 1 above)
Comment (jack-mo) [reply .eslintrc.json]: space invader
please deleted unneed new line

**Comment by AdrianMachado** [comment]
LGTM :shipit:

**Review by jack-mo — APPROVED** [review]
LGTM

**Review by AdrianMachado — APPROVED** [review]

**Pull request merged**